popsycle/data/pbh_config.yaml

```diff
@@ -0,0 +1,9 @@
+# Example configuration parameters for PBHs
```

Comment:
Note that many of these config parameters are defunct given the new method of sampling PBHs in a light cone. We should update the config file to reflect the new parameters.

Reply:
Already done. Will be in next push.

Reply:
Are you saying to get rid of r_vir and c, and to just have r_s as an input? r_s = r_vir/c so I kind of liked how we let the user choose those values. Just because we are using the McM17 paper for these values, doesn't mean everybody will.

Reply:
Please see my associated comment in the synthetic.py section.